lib/mapper.js

```diff
@@ -52,6 +49,7 @@ exports.page = function(page, settings){
 
 exports.screen = function(screen, settings){
   var result = createCommonGAForm(screen, settings);
```

Comment:
@ndhoule The tests work with and without hostname (dh) and path (dp). I'm basically erring on the side of caution (i.e. not breaking our server-side GA integration) by removing dh and dp since they don't logically make sense in the context of a screen call. I can leave them in if you think that's best.

So, do you 1) want me to leave dh and dp in for screen calls, 2) extract the pageview stuff to a separate helper, or 3) leave it as is? Also, I fixed the dot access issue.